build-aux/org.gnome.PortalTest.Qt6.json

```diff
@@ -1,7 +1,7 @@
 {
     "app-id": "org.gnome.PortalTest.Qt6",
     "runtime": "org.kde.Platform",
-    "runtime-version": "6.5",
+    "runtime-version": "6.6",
```

Comment:
not sure if that can be done but current is 6.7. FYI.

Reply:
This commit was part of #161 so I updated it there before merging. This PR is now up-to-date with that.